_component_design/buttons.md

```diff
@@ -14,203 +14,194 @@ last-modified: 2017-08-17
+
+- Weights:   
+    - Primary
+    - Secondary
+    - Teritiary
+- Sizes:
+    - Large
+    - Standard
+    - Compact
 
 {% endcolumn %}
-{% column right:"hxCol" %}
-{% figure [caption:"Secondary button sizes"] %}
-![]({{site.cdn_url}}/img/components/Buttons/buttons-secondary-sizes.svg){:width="100%"}
-{% endfigure %}
-{% endcolumn %}
-{% column right:"hxCol" %}
-{% figure [caption:"Secondary button states"] %}
-![]({{site.cdn_url}}/img/components/Buttons/buttons-secondary-actions.svg){:width="100%"}
+
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+{% figure [caption:"The three button weights"] [class:"image bg-light border"] %}
+![]({{site.url}}/assets/images/components/inputs-and-controls/buttons/buttons-simple.svg){:width="100%"}
 {% endfigure %}
 {% endcolumn %}
 </div>
 
-# Tertiary Buttons
-
+### Spacing and specs
+{: .hxSubSectionTitle}
 <div class="hxRow">
-{% column left:"hxCol" %}
--   Tertiary buttons can be used in conjunction with the primary button or with
-    a primary and secondary button.
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+For most desktop use cases, you should use a standard button size.   
 
--   Tertiary buttons should be used to deemphasize the action in comparison
-    with the actions of primary or secondary buttons.
+Large buttons should be used in mobile designs to accommodate the ideal touch target size. 
 
--   For example, use a tertiary button for "cancel" and a primary button to
-    emphasize "submit".
+Compact buttons should be used when you have severe space constraints, e.g. inside of a table or in a dashboard widget.
 
 {% endcolumn %}
-{% column right:"hxCol" %}
-{% figure [caption:"Tertiary button sizes"] %}
-![]({{site.cdn_url}}/img/components/Buttons/buttons-tertiary-sizes.svg){:width="100%"}
-{% endfigure %}
-{% endcolumn %}
-{% column right:"hxCol" %}
-{% figure [caption:"Tertiary button states"] %}
-![]({{site.cdn_url}}/img/components/Buttons/buttons-tertiary-actions.svg){:width="100%"}
+
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+{% figure [caption:"Button spacing specs"] [class:"image bg-light border"] %}
+![]({{site.url}}/assets/images/components/inputs-and-controls/buttons/buttons-specs.svg){:width="100%"}
 {% endfigure %}
 {% endcolumn %}
 </div>
 
-# Dropdowns
-
-Dropdowns inject additional actions into a primary, secondary, or tertiary
-button.
-
+### Primary buttons - simple
+{: .hxSubSectionTitle}
 <div class="hxRow">
-{% column left:"hxCol" %}
--   Inherits and extends any of the three button weights
-
--   Follows the same styles and states as the button weight which it extends
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
 
--   Attempt to limit to a maximum of 8 items. If more than 8 items are needed,
-    consider splitting a subset of actions into another location on the page.
+- Use the primary action button to prompt the user to take action
```

Comment:
Full stop after each sentence (.)
Applies throughout document

Reply:
Looked over document for all instances of a UL and ensured full stops were present.